napari_allencell_annotator/_tests/utils/test_image_utils.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+import numpy as np
+import pytest
+
+from napari_allencell_annotator.util.image_utils import ImageUtils
+
+
+def test_get_dask_data_tiff() -> None:
+    # ACT
+    test_image = ImageUtils(
+        Path("napari_allencell_annotator/_tests/assets/image_types/test_img.ome.tiff")
+    ).get_dask_data()
+
+    # ASSERT
+    assert np.all(test_image == np.array([[[0, 0, 0], [0, 0, 0]], [[0, 0, 0], [0, 0, 0]]]))
+
+
+def test_get_dask_data_tif() -> None:
+    # ACT
+    test_image = ImageUtils(
+        Path("napari_allencell_annotator/_tests/assets/image_types/test_img.ome.tif")
+    ).get_dask_data()
+
+    # ASSERT
+    assert np.all(test_image == np.array([[[0, 0, 0], [0, 0, 0]], [[0, 0, 0], [0, 0, 0]]]))
+
```

Comment:
can you just loop over all the files in the test images directory instead

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/77/commits/67343d5d68da648183d7fe0ff3ded119833eaf9f